lib/controller.js

```diff
@@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
         //load view
         lnCmsClientService.getView(endpoint, params)
           .then(function(response) {
-            if (response.data.length > 0)
+            if (response.data.length > 0) {
```

Comment:
That’s responsibility of the template.
 
> On Apr 4, 2016, at 10:00 PM, Adam Fenton <notifications@github.com> wrote:
> 
> In lib/controller.js <https://github.com/moxie-leean/ng-cms/pull/6#discussion_r58480948>:
> 
> > @@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
> >          //load view
> >          lnCmsClientService.getView(endpoint, params)
> >            .then(function(response) {
> > -            if (response.data.length > 0)
> > +            if (response.data.length > 0) {
> How does it know which property of the response object is the array?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/ng-cms/pull/6/files/bfcc4884b2934af2d9d2f11fa483402048416224#r58480948>